examples/streaming/index.tsx

```diff
@@ -12,26 +11,37 @@ async function runStreamingWithChildrenExample() {
 
   console.log("\n📝 Non-streaming version (waiting for full response):");
   await gsx.execute<string>(
-    <ChatCompletion prompt={prompt}>
-      {async (response: string) => {
-        await setTimeout(0);
-        console.log(response);
-      }}
-    </ChatCompletion>,
+    <OpenAIProvider apiKey={process.env.OPENAI_API_KEY}>
+      <ChatCompletion
+        model="gpt-4o-mini"
+        messages={[{ role: "user", content: prompt }]}
+      >
+        {async (response: string) => {
+          await setTimeout(0);
```

Comment:
Why did we ever have `await setTimeout(0) in this example? 

Likely a leftover from when this example just returned a string, no? Can we remove it?

Reply:
Yeah just left over from needing proof that async worked.